src/pages/contacts/contact-list/contact-list.ts

```diff
@@ -107,9 +107,22 @@ export class ContactListPage {
     this.profile = this.userDataProvider.currentProfile;
     this.network = this.userDataProvider.currentNetwork;
 
-    this.addresses = lodash(this.profile.contacts).mapValues('name').transform((result, key, value) => {
+    this.addresses = this.sortByName(lodash(this.profile.contacts).mapValues('name').transform((result, key, value) => {
       result.push({ index: value, value, key });
-    }, []).value();
+    }, []).value());
+  }
+
+  sortByName(contacts) {
+    const byName = function(a, b) {
+      if (a.key < b.key) {
+        return -1;
+      }
+      if (a.key > b.key) {
+        return 1;
+      }
+      return 0;
+    };
+    return contacts.sort(byName);
```

Comment:
this whole thing can probably be simplified by changing it so this:

```
    this.addresses = lodash(this.profile.contacts).mapValues('name').transform((result, key, value) => {
      result.push({ index: value, value, key });
    }, []).value().sort((one, two) => one.key.localeCompare(two.key));
```

Reply:
Cool. I did not know about the localeCompare and your solution is much more elegant 😄